src/org/nfrac/comportex/cells.cljc

```diff
@@ -1078,19 +1099,16 @@
   (predictive-cells [_]
     (when (== (:timestep state)
               (:timestep distal-state))
-      (set/union (:pred-cells distal-state)
-                 (:pred-cells apical-state))))
+      (:pred-cells distal-state)))
```

Comment:
I wonder if eventually we'd want `predictive-cells` to show only the cells that will become active if its column is activated. In other words, it'd account for apical connections and the dominance margin.